aries-site/src/examples/components/namevaluelist/Examples/NameValueListEditExample.js

```diff
@@ -0,0 +1,140 @@
+import React, { useContext, useState } from 'react';
+import { ThemeContext } from 'styled-components';
+import {
+  Anchor,
+  Box,
+  Button,
+  Form,
+  FormField,
+  Heading,
+  MaskedInput,
+  NameValueList,
+  NameValuePair,
+  TextInput,
+  Text,
+} from 'grommet';
+import { profileData } from '../data';
+
+export const NameValueListEditExample = () => {
+  const theme = useContext(ThemeContext);
+  const [currentData, setCurrentData] = useState(profileData);
+  const [tempData, setTempData] = useState(currentData);
+  const [edit, setEdit] = useState(false);
+
+  return (
+    <Box gap="medium">
+      <Box justify="between" width="large" direction="row">
+        <Heading level={2} size="small" margin="none">
+          Profile Details
+        </Heading>
+        {!edit && (
+          <Box justify="center">
+            <Button
+              alignSelf="center"
+              secondary
+              label="Edit Profile"
+              onClick={() => setEdit(true)}
+            />
+          </Box>
+        )}
+      </Box>
+      {!edit ? (
+        <NameValueList pairProps={{ direction: 'column' }}>
+          {Object.entries(currentData).map(([name, value]) => (
+            <NameValuePair
+              name={<Text {...theme.formField.label}>{name}</Text>}
+            >
+              {name === 'Email' ? (
+                <Anchor label={value} href={`mailto:${value}`} />
+              ) : (
+                <Text {...theme.global.input.font}>{value || '--'}</Text>
```

Comment:
Design calls for an 'xxsmall' spacing between label and value.

Additionally, design calls for a 'medium' gap when in column. Seems like we need to adjust the theme to address?

Reply:
Thoughts?

Reply:
That was my thinking on both as well.

Reply:
looking making these changes now will tag both of you once its up

Reply:
PR to fix the spacing can be found in these 2 PRs
https://github.com/grommet/grommet/pull/5788
https://github.com/grommet/grommet/pull/5789